btsieve/config/bitcoin_only.toml

```diff
@@ -1,10 +1,6 @@
 [bitcoin]
+network = "testnet"
```

Comment:
Why testnet? I haven't looked into why your tests are failing but this should be regtest no?

Reply:
irrelevant at this point, but changed to regtest :)